uuid.go

```diff
@@ -294,3 +294,15 @@ func DisableRandPool() {
 	poolMu.Lock()
 	poolPos = randPoolSize
 }
+
+// UUIDs is a slice of UUID types
+type UUIDs []UUID
+
+// Strings returns a slice of the string form of each UUID in the received slice of UUIDs
```

Comment:
// Strings returns a string slice containing the string form of each UUID in uuids.